packages/camera/camera/ios/Classes/FLTSavePhotoDelegate.m

```diff
@@ -3,37 +3,41 @@
 // found in the LICENSE file.
 
 #import "FLTSavePhotoDelegate.h"
+#import "FLTSavePhotoDelegate_Test.h"
+
+@interface FLTSavePhotoDelegate ()
+/// The file path for the captured photo.
+@property(readonly, nonatomic) NSString *path;
```

Comment:
```suggestion
@property(readonly, nonatomic, copy) NSString *path;
```

Reply:
oh this is already `readonly` so i leave out `copy`